scripts/deploy

```diff
@@ -118,18 +114,19 @@ if ! $mock_ctrl; then
   sed -i 's/DISABLE_HW = True/DISABLE_HW = False/' ../amplipi/rt.py
 fi
 
-if $dev; then
-  old_version=$(poetry version -s)
-  # git_info="$(git describe --always --dirty)-$(git rev-parse --abbrev-ref HEAD)"
-  # TODO: write extra git info to the pyproject.toml file since poetry complains about non-standard versions
-  poetry version prerelease
-fi
+# use a new version for the build but hide it from git since it isn't a real release
+old_version=$(poetry version -s)
+# parse the git info into: VERSION+GIT_HASH-BRANCH_NAME-DIRTY?
+# git describe gets us close to this: VERSION-NUM_COMMITS_SINCE-GIT-HASH-DIRTY?
+#  but we need to change it a bit to remove NUM_COMMITS and add BRANCH_NAME
+git_info=$(git describe --always --tags --dirty --match '*.*.*' | sed -E s/-[0-9]*-*/+/ | sed s/-/-$(git rev-parse --abbrev-ref HEAD)-/)
```

Comment:
This gives me an empty string. Did you mean
```suggestion
git_info=$(git describe --always --tags --dirty --match '*.*.*' | sed "s/-[0-9]*-*\(.*\)/+\1-$(git rev-parse --abbrev-ref HEAD)/")
```
Neither yours nor mine adds DIRTY, not sure what you wanted there.

Reply:
Turns out your version works when dirty, mine works when clean. TODO: make it work in both cases!